docs/development/extensions-contrib/spectator-histogram.md

```diff
@@ -0,0 +1,453 @@
+* With a single extra metric column of type `quantilesDoublesSketch` ingesting the `added` column
+
+Spectator histograms average just 6 extra bytes per row, while the data-sketch
+adds 48 bytes per row. This is an 8 x reduction in additional storage size.
+![Comparison of datasource sizes in web console](../../assets/spectator-histogram-size-comparison.png)
+
+As rollup improves, so does the size saving. For example, ingesting the wikipedia data
+with day-grain query granularity and removing all dimensions except `countryName`,
+we get to a segment that has just 106 rows. The base segment is 87 bytes per row,
+adding a single `spectatorHistogram` column adds just 27 bytes per row on average vs
+`quantilesDoublesSketch` adding 255 bytes per row. This is a 9.4 x reduction in additional storage size.
+Storage gains will differ per dataset depending on the variance and rollup of the data.
+
+## Background
+[Spectator](https://netflix.github.io/atlas-docs/spectator/) is a simple library
+for instrumenting code to record dimensional time series data.
+It was built, primarily, to work with [Atlas](https://netflix.github.io/atlas-docs/).
+Atlas was developed by Netflix to manage dimensional time series data for near
+real-time operational insight.
+
+With the [Atlas-Druid](https://github.com/Netflix-Skunkworks/iep-apps/tree/main/atlas-druid)
+service, it's possible to use the power of Atlas queries, backed by Druid as a
+data store to benefit from high-dimensionality and high-cardinality data.
+
+SpectatorHistogram is designed for efficient parallel aggregations while still
+allowing for filtering and grouping by dimensions. 
+It provides similar functionality to the built-in data-sketch aggregator, but is
+opinionated and optimized for typical measurements of cloud services and web-apps.
+Measurements such as page load time, transferred bytes, response time, request latency, etc.
+Through some trade-offs we're able to provide a significantly more compact
+representation with the same aggregation performance and accuracy as
+data-sketches (depending on data-set, see limitations below).
+
+## Limitations
+* Supports positive long integer values within the range of [0, 2^53). Negatives are
+coerced to 0.
+* Decimals are not supported.
+* 276 fixed buckets with increasing bucket widths. In practice, the observed error of computed percentiles is in the range (0.1%, 3%). See [Bucket Boundaries](#histogram-bucket-boundaries) for the full list of bucket boundaries.
+* DruidSQL queries are yet not supported. You must use native Druid queries.
+* Vectorized queries are yet not supported.
+
+> If any of these limitations are a problem, then the data-sketch aggregator
+is most likely a better choice.
+
+## Functionality
+The SpectatorHistogram aggregator is capable of generating histograms from raw numeric
+values as well as aggregating/combining pre-aggregated histograms generated using
```

Comment:
```suggestion
values as well as aggregating or combining pre-aggregated histograms generated using
```
Use "or" or "and" instead of "/" which can be ambiguous